Automattic/jetpack
# Forms: Update patterns modal default view mode

## Proposed changes:

* Updates the Form patterns modal default view mode to Grid

### Other information:


## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:

* Create a post and add a Form block
* Click the "Explore Forms Patterns" button
* Check if the modal opens with the grid view mode selected by default
